src/main/java/de/funfried/netbeans/plugins/external/formatter/ui/options/ExternalFormatterPanel.form

```diff
@@ -122,7 +122,7 @@
           <ResourceString bundle="de/funfried/netbeans/plugins/external/formatter/ui/options/Bundle.properties" key="ExternalFormatterPanel.btnDonate.toolTipText" replaceFormat="org.openide.util.NbBundle.getMessage({sourceFileName}.class, &quot;{key}&quot;)"/>
         </Property>
         <Property name="cursor" type="java.awt.Cursor" editor="org.netbeans.modules.form.editors2.CursorEditor">
-          <Color id="Handcursor"/>
+          <Color id="Default Cursor"/>
```

Comment:
Why did you change that? Could you please revert it?

Reply:
No idea how that happened. Netbeans does not even show that FORM in the file list. I had to revert it manually from the terminal. Sorry for that unjustified modification.

Reply:
Same here: Did you already push your changes?